src/misosoupy/psychopy_present_item_list.py

```diff
@@ -0,0 +1,346 @@
+                        fillColor=None,
+                        lineColor=shapeLineColor,
+                        lineWidth=3.0,
+                    )
+                )
+        else:
+            allScreenWords.append(
+                visual.TextStim(
+                    win,
+                    text=currItemText,
+                    pos=allWordPosValues[iItem],
+                    color=textColor,
+                    height=currItemHeight,
+                    alignText="Left",
+                )
+            )
+            allBoxes.append(
+                visual.ShapeStim(
+                    win,
+                    vertices=((-0.5, -0.3), (-0.5, 0.3), (0.5, 0.3), (0.5, -0.3)),
+                    pos=allSquarePosValues[iItem],
+                    size=(squareSize * 0.5, squareSize * 1.5),
+                    opacity=100,
+                    fillColor=None,
+                    lineColor=shapeLineColor,
+                    lineWidth=3.0,
+                )
+            )
+
+    # # Prep Continue Button
+    instructTxt1 = visual.TextStim(
+        win, text=instr1, pos=(-0.7, 0.1), color=textColor, height=0.09, wrapWidth=6
+    )
+    instructTxt2 = visual.TextStim(
+        win, text=instr2, pos=(-0.7, 0.05), color=instr2color, height=0.09, wrapWidth=6
+    )
+    instructTxt3 = visual.TextStim(
+        win, text=instr3, pos=(-0.7, -0.85), color=textColor, height=0.09, wrapWidth=6
+    )
+    continueText = visual.TextStim(
+        win,
+        text="CONTINUE", #"Click here to continue",
+        pos=(0.7, -0.85),
+        color=screenColor,
+        height=0.08,
+    )
+    backText = continueText
+    exitShape = visual.ShapeStim(
+        win,
+        vertices=((-0.5, -0.3), (-0.5, 0.3), (0.5, 0.3), (0.5, -0.3)),
+        pos=(.75, 0.9),
+        size=(0.35, 0.2),
+        opacity=100,
+        fillColor=screenColor,
+        lineColor=shapeLineColor,
+        lineWidth=4.0,
+        name="exitShape",
+    )
+    exitText = visual.TextStim(
+        win,
+        text="EXIT", #"Click here to continue",
+        pos=(.75, 0.9),
+        color=textColor,
+        height=0.08,
+    )
+    continueShape = exitShape
+    backShape = exitShape
+    
+    # If page has been completed, re-present choices
+    if len(initSquares) > 0:
+        for i in range(len(initSquares)):
+            if initSquares[i]==1:
+                allBoxes[i]=visual.ShapeStim(win, vertices=((-.5, -.3), (-.5, .3), (.5, .3), (.5, -.3)), pos=allSquarePosValues[i], size=(squareSize*.5,squareSize*1.5), opacity=100, fillColor=shapeLineColor, lineColor=shapeLineColor,lineWidth=3.0)
+        itemsChosen = initSquares
+    else:
+        itemsChosen = [0 for i in range(len(currPageItems))]
+
+    Mouse = event.Mouse(win=win, visible=True)
+    Mouse.clickReset()
+    event.clearEvents()
+    previousMouseDown = False
+
+    itemClicked = False
+    continueChosen = False
+    backChosen = False
+    # itemsChosen = [0 for i in range(len(currPageItems))]
```

Comment:
commented out code can be confusing, ideally delete (bad habit of mine too, but I try to remind myself that with git I can always go back to the old code)